### PR TITLE
changed ruby sponsorship status to available

### PR DESCRIPTION
**Pull request by spinecone**

### CODE REVIEW

**Comment by spinecone** [comment]
For #91

File: sponsors.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Contact us!
 <a href="mailto:info@rubyforgood.org?subject=Ruby Sponsorship" class="btn btn-sm btn-primary">1 Available</a>
 </div-->
```
Comment (adunkman) [inline sponsors.md]: I think you can axe this comment now, too! 💯 😃

Comment (spinecone) [inline sponsors.md]: 💣 💥 💥

**Comment by adunkman** [comment]
Nice nice! 👍 🚀 🚢 ⛵ 

sloth kittenssss

**Pull request merged**